frontend/components/transaction/SurplusAuctionBidTransactionTable.vue

```diff
@@ -0,0 +1,138 @@
+        </div>
+        <div class="flex justify-between">
+            <div>Current Highest Bid</div>
+            <div>
+                <FormatCurrency :value="auction.bidAmountMKR" currency="MKR" />
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Lowest Next Bid</div>
+            <div>
+                <FormatCurrency :value="lowestNextBid" currency="MKR" />
+            </div>
+        </div>
+        <div class="flex justify-between items-center">
+            <div>The Amount to Bid</div>
+            <div class="flex w-1/2 items-center space-x-2 justify-end -mr-1">
+                <div class="w-full flex-shrink-0">
+                    <bid-input
+                        :transaction-bid-amount.sync="transactionBidAmount"
+                        :minimum-bid-dai="lowestNextBid"
+                        :disabled="!isActive"
+                        auction-type="surplus"
+                    />
+                </div>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Combined Transaction Fees</div>
+            <div>Unknown</div>
+        </div>
+        <div class="flex justify-between">
+            <div>Price on Uniswap</div>
+            <div>Unknown</div>
+        </div>
+        <div class="flex justify-between font-bold">
+            <div>Price after the bid</div>
+            <div>
+                <format-currency
+                    v-if="unitPriceAfterBid && isActive && !isBidAmountNaN"
+                    :value="unitPriceAfterBid"
+                    currency="MKR"
+                />
+                <div v-else>
+                    <span class="opacity-50">Unknown</span>
+                    <span class="font-bold">MKR</span>
+                </div>
+                <span class="flex">per DAI</span>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import type { SurplusAuction } from 'auctions-core/src/types';
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import BidInput from '../utils/BidInput.vue';
+import TimeTill from '~/components/common/TimeTill.vue';
+import FormatCurrency from '~/components/utils/FormatCurrency.vue';
+
+export default Vue.extend({
+    components: {
+        TimeTill,
+        FormatCurrency,
+        BidInput,
+    },
+    props: {
+        auction: {
+            type: Object as Vue.PropType<SurplusAuction>,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            transactionBidAmount: undefined as BigNumber | undefined,
+        };
+    },
+    computed: {
+        isActive(): boolean {
+            return this.auction.state === 'just-started' || this.auction.state === 'have-bids';
+        },
+        isBidAmountNaN(): boolean {
+            return !!this.transactionBidAmount?.isNaN();
+        },
+        lowestNextBid(): BigNumber {
+            return this.auction.bidAmountMKR.multipliedBy(1.1);
```

Comment:
Is this the correct function to calculate the lowest next bid? As I was gone for two weeks, I am not up-to-date on this topic 😄

Reply:
Actually, it is not; it is a temporary calculation until the actual function is implemented 😄

Reply:
In that case, please add a TODO comment, to make sure we catch this before merging 😄 

Example:
```js
// TODO: Implement correct calculation logic for the next possible lowest bid
```